access_list_test.go

```diff
@@ -1,315 +0,0 @@
-package jwt
```

Comment:
@njones , please resync with the latest branch. By the way, right after we merge this one, I am changing `master` branch to `main`.

Reply:
👍 - Would you also consider changing the `"master": true` and items that rely on having a `master` value to either `main` or `default` as well?

Reply:
> 👍 - Would you also consider changing the "master": true fields in pool.go and items that rely on having a master value to either main or default as well?

@njones, yes, I would change it to "primary" after this PR. Also, the "Masters" will become "PrimaryInstances". The thing is that "master" appears in other plugins "caddy-auth-{jwt,portal,saml}". I will be changing all of the in one shot. They need to be in sync.